CHANGELOG.txt

```diff
@@ -51,6 +51,9 @@ All notable changes to this project starting with the 0.6.0 release will be docu
 - `site jobs` will be removed in v0.7.0 (#412)
 - `site notifications` will be removed in v0.7.0 (#412)
 
+###Removed
+- Removed --branch-create flag from `sites code`
```

Comment:
Oh no!!!! Merging this changelog has been the bane of my afternoon! I'll add it in there